doorkeeper-i18n.gemspec

```diff
@@ -17,4 +17,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency "rake"
   s.add_development_dependency "rspec"
   s.add_development_dependency "i18n-spec", "~> 0.6.0"
+  s.add_development_dependency "railties", ">= 0"
```

Comment:
We don't need to add this dependency here. `doorkeeper-i18n` depends on `doorkeeper`, and that project depends on `railties`.